slides.md

```diff
@@ -507,7 +508,7 @@ export default async function customer(fastify) {
 <div class="dense">
 
 - Run the server for step 4 (`cd src/a04-insecure-design`, `npm start`)
-- In Postman, run the query for `A04: Buy product`. Observe the data for `success: true` being returned
+- From the browser at `http://localhost:3006`, run the query for `A04: Buy product`. Observe the data for `success: true` being returned
```

Comment:
in this case since it's about spam clicking the button, it would be nice to have an animation or change that shows that the button is being clicked otherwise it doesnt feel responsive